test/community/label_propagation.jl

```diff
@@ -1,9 +1,10 @@
 @testset "Label propagation" begin
     n = 10
     g10 = complete_graph(n)
-    for g in testgraphs(g10)
+    @testset "$g" for g in testgraphs(g10)
         z = copy(g)
         for k = 2:5
+            Random.seed!(0)
```

Comment:
let's not do this here. We still need to discuss a comprehensive solution - label propagation isn't the only place we get random failures.